src/components/inputs/date/date.component.tsx

```diff
@@ -177,7 +177,7 @@ const DateField: React.FC<FormFieldProps> = ({ question, onChange, handler, prev
                   // be triggered as opposed to the former handler that only gets triggered at runtime.
                   onChange={(e) => onDateChange([dayjs(e.target.value, placeholder.toUpperCase()).toDate()])}
                   disabled={question.disabled}
-                  invalid={isFieldRequiredError && errors.length > 0}
+                  invalid={isFieldRequiredError || errors.length > 0}
```

Comment:
@kajambiya do we still need to keep track of the `isFieldRequiredError` state?

Reply:
Actually no.